rskj-core/build.gradle

```diff
@@ -160,6 +160,7 @@ dependencies {
     testCompile "org.powermock:powermock-core:${powermockitoVersion}"
     testCompile "org.powermock:powermock-module-testng:${powermockitoVersion}"
     testCompile "org.powermock:powermock-module-junit4:${powermockitoVersion}"
+    testCompile "org.powermock:powermock-api-mockito:${powermockitoVersion}"
```

Comment:
Missing org.powermock:powermock-api-mockito and sub dependencies hashes for dependency verification

Reply:
Will do, cheers